app.json

```diff
@@ -25,6 +25,7 @@
     },
     "web": {
       "favicon": "./assets/favicon.png"
-    }
+    },
+    "fonts": "./assets/fonts/JetBrains-Mono-Regular.ttf'"
```

Comment:
This will become unnecessary if you use `expo-google-fonts` as suggested in my previous comments